src/user/user.entity.ts

```diff
@@ -1,10 +1,20 @@
-import { Column, Entity, PrimaryGeneratedColumn } from "typeorm";
+import { Column, Entity, Unique, PrimaryGeneratedColumn } from "typeorm";
 
 @Entity()
+@Unique(["email"])
 export class User {
 	@PrimaryGeneratedColumn()
 	id: number;
 
 	@Column("text", { name: "first_name" })
```

Comment:
try type "varchar" with length for speed optimization

Reply:
only for email column, or for all columns ?